assets/lang/sv-SE.json

```
@@ -523,7 +524,7 @@
   "negative": "Negativt",
   "failed": "Misslyckades",
   "pending": "Väntar",
-  "please-add-test-below": "Om du har testats för covid-19, lägg till testet nedan. Du kan redigera dessa tester när som helst i framtiden.",
+  "please-add-test-below": "Om du har gjort ett covid-19-test någon gång, lägg till det nedan. Detta innefattar PCR-, antigen- och antikroppstester. Du kan redigera dessa när som helst i framtiden.",
```

```suggestion
  "please-add-test-below": "Om du har gjort ett covid-19-test någon gång, lägg till det nedan. Detta innefattar PCR-tester och serologiska tester. Du kan redigera dessa när som helst i framtiden.",
```
Got a new context-based copy change in Swedish

---

Just edit the file!

---

It's quicker no?